articles/nodejs-frontend-or-backend/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/nodejs-frontend-or-backend/
+title: Node.js - Frontend or Backend?
+description:
+author: lalithnarayan-c
+date: 2020-08-19T00:00:00-15:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/engineering-education/nodejs-frontend-or-backend/hero.jpg
+    alt: front end and backend with Node.js
+---
+Let us start by understanding what Node.js is. Node.js is a runtime environment for executing server-side code with higher efficiency and it presents a larger bandwidth to handle large code payloads.
+<!--more-->
+
+1. Understand and clarify any doubt of whether Node.js is frontend or backend application
+2. List out various applications in both frontend and backend
+3. Understand how Node.js complements other technologies in the frontend and backend
+
+The choice of the language that Ryan Dahl, the creator of Node.js, used was the most popular programming language at the time, JavaScript.
```

Comment:
Please review this paragraph - it was a bit choppy and difficult to make out. 
Made minor changes where I could